lib/plugins/aws/lib/validate.js

```diff
@@ -15,18 +15,27 @@ module.exports = {
 
     this.options.stage = this.provider.getStage();
     this.options.region = this.provider.getRegion();
+
+    if (this.serverless.processedInput && this.serverless.processedInput.commands) {
```

Comment:
This `if` is superfluous, can be removed I believe

Reply:
Not quite. Without it, a few of the tests were broken.

Reply:
This is an issue of tests setup (as those objects are guaranteed to be there in normal `serveless` evaluation).

Do only tests in context of `validate.test.js` fail, or there's more?

Reply:
There was more. I looked the tests over and tweaked their setups...reissued the PR to reflect the changes.